server/snapshot/snapshot.go

```diff
@@ -1,11 +1,11 @@
 package snapshot
 
 import (
-	"encoding/json"
-
 	"github.com/Sirupsen/logrus"
 
+	"github.com/docker/go/canonical/json"
```

Comment:
:+1:  awesome!  I'm pretty excited about all of the tests you've added, but I'll let you know if I think of anything else